examples/plot_simulate_gamma.py

```diff
@@ -33,30 +33,41 @@
 net = Network(params)
 dpls = simulate_dipole(net, n_trials=1)
 
+###############################################################################
+# The network requires some time to reach steady state. Hence, we omit the
+# first 50 ms in our time-frequency analysis.
+
+tstart = 50
+mask = dpls[0].times > tstart
+times = dpls[0].times[mask]
+data = dpls[0].data['agg'][mask]
+
 ###############################################################################
 # We can plot the time-frequency response using MNE
 import numpy as np
 import matplotlib.pyplot as plt
-from mne.time_frequency import tfr_array_multitaper
+from mne.time_frequency import tfr_array_morlet
 
 fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6))
-dpls[0].plot(ax=axes[0], layer='agg', show=False)
+axes[0].plot(times, data)
```

Comment:
I was in two minds whether to add `tstart` and `tstop` to the plotting routine or just directly use matplotlib to plot. I settled with the later ... the utility of a plotting routine for Dipole object seems a bit limited to me unless we come up with more use cases

Reply:
Are you thinking of just removing the plot() methods from Dipole? I agree it's fairly unnecessary considering most people are capable of making a line plot. 

Now there's certainly a case to be made for more visualization options (multi-trial with mean dipole overlaid for example), but that would be better suited for `viz.py`.

Reply:
right, so perhaps we need a `plot_dipole` function? Still it's rather straightforward to do with 2 lines of Python code, but maybe we can add functionality later.